keymaster/ta/operations.c

```diff
@@ -223,21 +227,24 @@ keymaster_error_t TA_start_operation(
 				const keymaster_block_mode_t mode,
 				const uint32_t mac_length,
 				const keymaster_digest_t digest,
-				const keymaster_blob_t nonce)
+				const keymaster_blob_t nonce,
+				uint8_t *key_id)
 {
 	keymaster_error_t res = TA_try_start_operation(op_handle, key, min_sec,
 							operation, purpose,
 							digest_op, do_auth,
 							padding, mode,
-							mac_length, digest, nonce);
+							mac_length, digest,
+							nonce, key_id);
 	if (res != KM_ERROR_OK) {
 		res = TA_kill_old_operation();
 		if (res == KM_ERROR_OK) {
 			res = TA_try_start_operation(op_handle, key, min_sec,
 							operation, purpose,
 							digest_op, do_auth,
 							padding, mode,
-							mac_length, digest, nonce);
+							mac_length, digest,
```

Comment:
Please line up indentation with leading parenthesis (even the line you haven't touched).